manifests/init.pp

```diff
@@ -369,6 +369,8 @@
     }
 
     class { 'pulpcore':
+      allowed_import_path       => $allowed_import_path,
+      allowed_export_path       => $allowed_export_path,
```

Comment:
```suggestion
      allowed_import_path       => $pulpcore_allowed_import_path,
      allowed_export_path       => $pulpcore_allowed_export_path,
```

This keeps it clearer that we have 2 sets of parameters: `pulpcore::allowed_{import,export}_path` which is in the puppet-pulpcore module, and `foreman_proxy_content::pulpcore_allowed_{import,export}_path` here in puppet-FPC. The second pair only is what will be directly accessible by foreman-installer, so including pulpcore in the parameter name makes it easy to group that param with others that affect the pulpcore class.

Reply:
thanks for the help! :) Updated